packages/stark-ui/assets/styles/_variables.scss

```diff
@@ -1,12 +1,49 @@
-$border-radius: 2px;
+$color-theme: (
+  accent: (
+    color: #8f0039,
+    contrast: $light-primary-text
+  ),
+  alert: (
+    color: #d32f2f,
+    contrast: $light-primary-text
+  ),
+  alt: (
+    color: #323232,
+    contrast: $light-primary-text
+  ),
+  neutral: (
+    color: #bfbfbf,
+    contrast: $light-primary-text
+  ),
+  primary: (
+    color: #0076c8,
+    contrast: $light-primary-text
+  ),
+  success: (
+    color: #4caf50,
+    contrast: $light-primary-text
+  ),
+  warning: (
+    color: #ff9800,
+    contrast: $light-primary-text
+  ),
+  white-transp: (
+    color: rgba(255, 255, 255, 0.87),
+    contrast: $dark-primary-text
+  )
+);
+
+@if variable-exists(stark-color-theme) {
+  $color-theme: map-merge($color-theme, $stark-color-theme);
```

Comment:
"$stark-color-theme" ? Wasn't it renamed in the file above?

Reply:
That's indeed why I renamed above. We give the ability to provide a new map called $stark-color-theme, which contains a simple version of the colors (instead of the full palette). Well as soon as https://github.com/NationalBankBelgium/stark/pull/672 is merged, I will add the doc for that (https://github.com/NationalBankBelgium/stark/issues/679). 

Robby need this one to be merged to finish the language selector, that's why I have separate the doc :-)